pkg/clusteragent/admission/mutate/agent_sidecar/agent_sidecar.go

```diff
@@ -46,12 +46,21 @@ type Webhook struct {
 	operations        []admiv1.OperationType
 	namespaceSelector *metav1.LabelSelector
 	objectSelector    *metav1.LabelSelector
+	containerRegistry string
 }
 
 // NewWebhook returns a new Webhook
 func NewWebhook() *Webhook {
 	nsSelector, objSelector := labelSelectors()
 
+	var containerRegistry string
+
+	if config.Datadog.IsSet("admission_controller.agent_sidecar.container_registry") {
```

Comment:
Given that this is basically the same in the 3 webhooks, would it make sense to define a function in the common package and reuse it? Something like `func containerRegistry(specificConfigOpt string) string`?